applications/MappingApplication/mapping_application.h

```diff
@@ -142,6 +146,11 @@ class KratosMappingApplication : public KratosApplication {
 	///@}
 	///@name Protected member Variables
 	///@{
+  
+  // Needed for Volume Mapping
```

Comment:
This will involve a bigger change, as of now I only work with Conditions. Of course I could change large parts of the code to GeometricalObject, but I also use functions such as NumberOfConditions(), which makes it more difficult.
Therefore I thought creating the volume conditions is the easier solution.

I discussed it with pooyan already, we agreed that if the input submodelpart contains Elements, I work with Elements, if  it contains Conditions, I work with conditions, if it contains both I throw an error.

I will implement it in this way then